test/order_test.rb

```diff
@@ -114,10 +115,23 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      orders = Order.all
+      p orders
+
+      expect(orders.length).must_equal 100
+      orders.each do |o|
+        expect(o).must_be_kind of Order
+        expect(o.id).must_be_kind_of Integer
+        expect(o.products).must_be_kind_of Hash
+        expect(o.customer).must_be_kind_of Integer
```

Comment:
`o.customer` should be a `Customer`:

```suggestion
        expect(o.customer).must_be_kind_of Customer
```